src/cascadia/TerminalApp/AppKeyBindings.cpp

```diff
@@ -82,6 +82,7 @@ static const std::map<std::wstring_view, ShortcutAction> commandNames {
     { SWITCHTOTAB6_KEY, ShortcutAction::SwitchToTab6 },
     { SWITCHTOTAB7_KEY, ShortcutAction::SwitchToTab7 },
     { SWITCHTOTAB8_KEY, ShortcutAction::SwitchToTab8 },
+    { OPENSETTINGS_KEY, ShortcutAction::OpenSettings },

```

Comment:
aah, did we miss this in serialization as well?

Reply:
(not a hint that you should fix it; i'll file a followup issue)

Reply:
Serialization seems to be fine with this line added.

Reply:
Yikes that's my bad